Matricies/Matricies/Measurments.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.IO;
+using System.Threading.Tasks;
+using System.Timers;
+using System.Diagnostics;
+
+
+namespace Matricies
+{
+    public class Measurments
+    {
+            int numOfIterations;
+
+            string res_path;
+
+            string f_path;
+            string s_path;
+
+            public Measurments(int numIt, string path, string path_1, string path_2)
+            {
+                numOfIterations = numIt;
+
+                f_path = path_1;
+                s_path = path_2;
+                res_path = path;
+            }
+
+        /// <summary>
+        /// counts time of work of a given method 
+        /// </summary>
+        
+            long count_time(MyMatrix A, MyMatrix B, Func<MyMatrix, MyMatrix, MyMatrix> func)
+            {
+                Stopwatch timer = new Stopwatch();
+
+                timer.Start();
+                func(A, B);
+                timer.Stop();
+
+                return timer.ElapsedMilliseconds;
+            }
+
+        /// <summary>
+        /// compares to multiplications 
+        /// </summary>
+       
+            public void Comparison(int dim, StreamWriter sw)
+            {
+                long time_consistent = 0;
+                long time_async = 0;
+
+                long average_cons = 0;
+                long average_async = 0;
+
+                long variance_async = 0;
+                long variance_cons = 0;
+
+
+                for (int i = 0; i < numOfIterations; i++)
+                {
+
+                    MyMatrix A = MyMatrix.generator(dim, dim);
+                    MyMatrix B = MyMatrix.generator(dim, dim);
+
+                    try
+                    {
+                        long time = count_time(A, B, MyMatrix.mult_async_3); ;
+                        time_async += time;
+                        variance_async += time * time;
+                    }
+                    catch (Exception e)
```

Comment:
```suggestion
                    catch (Exception)
```
GitHub Actions ругается